zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,36 @@
+Receive Zulip notifications for your Uptime Robot monitors!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+    Make sure to specify the topic in the URL above. Otherwise, the
+    default topic `General` will be used.
+
+1. Go to **My Settings** on Uptime Robot, and select **Add Alert Contact**.
+   Set **Alert Contact Type** as webhook.
+
+1. Set **Friendly Name** to a name of your choice, such as `Zulip`. Set **URL to notify** to the
+   URL constructed above with a **&** added at the end. Select **Send as JSON (application/json)**
+   under **POST Value(JSON Format)**.
+
+1. Set **POST Value(JSON Format)** to
+
+
+   >{
+   >    "monitorID":"*monitorID*",
+   >    "monitorURL":"*monitorURL*",
+   >    "monitorFriendlyName":"*monitorFriendlyName*",
+   >    "alertType":"*alertType*",
+   >    "alertTypeFriendlyName":"*alertTypeFriendlyName*",
+   >    "alertDetails":"*alertDetails*",
+   >    "alertFriendlyDuration":"*alertFriendlyDuration*"
+   >}
```

Comment:
How did you design this format?

Reply:
In Uptime Robot webhook we have the flexibility to choose how the JSON should look like. I was trying to base my message on the e-mail sent by Uptime Robot and these were the minimum variables required for the replication. I also saw a [Microsoft Teams-Uptime Robot Integration](https://gist.github.com/taija/76c2e22cba7d07e476d0457b4e8cda53) and borrowed some elements from its JSON format.